doc/grants.md

```diff
@@ -0,0 +1,44 @@
+# Grants
+
+[More ideas here](https://intranet.uclouvain.be/en/myucl/administrations/adre/research-funding-opportunities.html)
+
+
+Contact person at UCL can help a lot: A.D.

```

Comment:
I think it is fine to put names, at least here these are public info and for us we will forget in a couple of days who A.D. is. It is probably better to not share emails (even though very likely public as well)

```suggestion
Contact person at UCL can help a lot: Anouk Distelman

```